packages/safety-suite-api/src/api/dashboardSink/fleetPerfomanceDashboard.ts

```diff
@@ -0,0 +1,335 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+
+interface Sort {
+  column: string;
+  direction?: string;
+}
+
+interface Query {
+  customerAlias: string;
+  groupIds?: string[];
+}
+
+interface TableQuery extends Query {
+  sort: Sort;
+  page?: number;
+  size?: number;
+}
+
+interface Model {
+  id: number;
+  employeeLabel: string;
+  customerAlias: string;
+  groupId: number;
+}
+
+// Accident
+interface AccidentCounts {
+  daysSincePreventable: number;
+}
+
+interface Accident extends Model {
+  recordNumber: number;
+  date: string;
+  terminalLabel: string;
+  type: string;
+  preventable: boolean;
+  daysSince: number;
+}
+
+// Development Plans
+
+interface DevelopmentPlanCounts {
+  activePlans: number;
+  tasksAssigned: number;
+  tasksPastDue: number;
+}
+
+interface DevelopmentPlan extends Model {
+  recordNumber: number;
+  planType: string;
+  planStatus: string;
+}
+
+// Enforcement
+
+interface EnforcementCounts {
+  inspections: number;
+  cleanInspections: number;
+  inspectionsWithViolations: number;
+}
+
+interface Enforcement extends Model {
+  recordNumber: number;
+  date: string;
+  csaPoints: number;
+  outOfService: boolean;
+}
+
+// Expiring Documents
+
+interface ExpirationCounts {
+  expired: number;
+  expiresIn30Days: number;
+  expiresIn60Days: number;
+}
+
+interface Expiration extends Model {
+  recordNumber: number;
+  expirationDate: string;
+  terminalLabel: string;
+  recordType: string;
+}
+
+// Injuries
+
+interface InjuryCounts {
+  daysSince: number;
+}
+
+interface Injury extends Model {
+  recordNumber: number;
+  date: string;
+  terminalLabel: string;
+  oshaReportable: boolean;
+  daysSince: number;
+}
+
+// Watch List
+
+type RiskScoreBuckets =
+  | '1'
+  | '11'
+  | '21'
+  | '31'
+  | '41'
+  | '51'
+  | '61'
+  | '71'
+  | '81'
+  | '91';
+
+type RiskScoreGraph = {[R in RiskScoreBuckets]: number};
+
+interface RiskScore extends Model {
+  terminalLabel: string;
+  riskScore: number;
+  planStatus: string;
+}
+
+const getSortString = (sort: Sort): string => {
+  return sort.direction ? `${sort.column},${sort.direction}` : sort.column;
+};
```

Comment:
With the change to an array above, this just becomes `return sorts.map(sort => ... current code ... )`